### PR TITLE
feat: Perform checks on VCs embedded inside VPs

### LINKED ISSUE #410: Perform checks on VCs embedded inside VPs

### PR DESCRIPTION
**Pull request by rolsonquadras**
closes #410 

Signed-off-by: Rolson Quadras <rolson.quadras@securekey.com>

### CODE REVIEW

File: pkg/restapi/vc/operation/operations.go [hunk 1]
```diff
@@ -1690,7 +1690,7 @@ func (o *Operation) parseAndVerifyVP(vpBytes []byte) (*verifiable.Presentation,
 			return nil, err
 		}
 		// verify if the credential in vp is valid
-		_, err = o.parseAndVerifyVCStrictMode(vcBytes)
+		err = o.validateCredentialProof(vcBytes, nil)
```
Comment (troyronda) [inline pkg/restapi/vc/operation/operations.go:1693]: I assume this is also in strict mode?

Comment (sudeshrshetty) [inline pkg/restapi/vc/operation/operations.go:1693]: saw this `// TODO figure out the process when vc has more than one proof`, I think we should verify all.

Comment (rolsonquadras) [reply pkg/restapi/vc/operation/operations.go:1693]: Yes, we need to implement that.

Comment (rolsonquadras) [reply pkg/restapi/vc/operation/operations.go:1693]: Yes.

Comment (rolsonquadras) [reply pkg/restapi/vc/operation/operations.go:1693]: this function internally calls parseAndVerifyVCStrictMode + proofPurpose validation.

Comment (troyronda) [reply pkg/restapi/vc/operation/operations.go:1693]: #412

**Pull request merged**